recipes/osfclient/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "osfclient" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "26a0d8f5611f50b8e3ff37a0ce37959e77fbc6941d180aecd9b4242e75d69823" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
```

Comment:
You should add [entry points](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points) here to match [the ones in `setup.py`](https://github.com/dib-lab/osf-cli/blob/0.0.3/setup.py#L52-L56). (Unfortunately you have to specify them in the recipe too.)